### DISCUSSION TITLE
一朵多雲有多重

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1561] [General]
我平常很喜歡看天空看雲，但看著看著我就突然很好奇雲的重量是多少，而查完資料後發現每天在天上飄的雲其實比我們想像中的還要重

每立方公尺的雲大概含有 0.5 克的水​
而一朵經典款棉花糖積雲大小約 1 立方公里​
掐指一算，它其實就有 50 萬公斤那麼重​
​
這個重量呢，相當於一千隻卡比獸​
五百顆中部粽、一百頭亞洲象、三條藍鯨​、一列自強號火車、或半截巴黎鐵塔​
在你頭頂上隨風而來又隨風而去

**Comment by Alex11392** [comment]
真的這樣算出來的結果好有趣，也讓我大吃一驚，竟然那麼重
